src/controllers/playback/video/index.js

```diff
@@ -1681,6 +1775,27 @@ export default function (view) {
         }
     });
 
+    nowPlayingPositionSlider.updateBubbleHtml = function(bubble, value) {
+        showOsd();
+
+        const item = currentItem;
+        let ticks = currentRuntimeTicks;
+        ticks /= 100;
+        ticks *= value;
```

Comment:
I don't see any reason to break this into separate lines.

```suggestion
        const ticks = currentRuntimeTicks * value / 100;
```